clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -719,6 +732,11 @@ public boolean matches(AbstractRequest body) {
 
     @Test
     public void testUpdateLastHeartbeatPollWhenCoordinatorUnknown() throws Exception {
+        GroupRebalanceConfig rebalanceConfig = buildRebalanceConfig(Optional.empty());
+        ConsumerCoordinator coordinator = buildCoordinator(rebalanceConfig,
```

Comment:
In many test cases now we don't seem to take advantage of initialization of `coordinator` in the `setup` method. Any reason why this changed?

Reply:
Sounds good, let me remove some of them